data/pathfinder/paizo/adventure_path/curse_of_the_crimson_throne/ap9/ap9_races.lst

```diff
@@ -7,5 +7,5 @@ SOURCELONG:Curse of the Crimson Throne, Chapter 3 - Escape from Old Korvosa	SOUR
 # For the offical Pathfinder RPG version of the rakshasa maharajah, use the maharajah rakshasa from the Pathfinder RPG Bestiary 3 dataset.
 
 # Race Name			Product Identity?	Starting Feats	Size		Move				Face/Space	Reach		Vision			Bonus to HP			Bonus to skill									Stat bonus																		Modify VAR						Weapon prof. bonus			Ability												Natural Attacks																				Damage Reduction	Monster Class Name and Starting Level	Main Race Type	Template		Challenge Rating	Source Page
-Rajput Ambari		NAMEISPI:YES				SIZE:H	MOVE:Walk,40		FACE:15	REACH:10	VISION:Darkvision (60')	BONUS:HP|CURRENTMAX|112												BONUS:STAT|STR|16	BONUS:STAT|DEX|2	BONUS:STAT|CON|0	BONUS:STAT|INT|0	BONUS:STAT|WIS|0	BONUS:STAT|CHA|-10	BONUS:VAR|AC_Natural_Armor|12|TYPE=Base							ABILITY:Special Ability|AUTOMATIC|Trample|Undead Servitor|War Stomp		NATURALATTACKS:Slam,Weapon.Melee.Natural,*2,2d6|Stamp,Weapon.Melee.Natural,*2,2d6	NATURALATTACKS:Gore,Weapon.Melee.Natural,*1,2d8	DR:10/slashing	MONSTERCLASS:Undead (Mindless):16		RACETYPE:Undead				CR:7			SOURCEPAGE:p.84
-Sikari Macaque Swarm				STARTFEATS:1	SIZE:T	MOVE:Walk,30,Climb,30	FACE:10	REACH:0	VISION:Low-Light Vision					BONUS:SKILL|Climb|16	BONUS:SKILL|Acrobatics|8|TYPE=Racial	BONUS:STAT|STR|-6	BONUS:STAT|DEX|10	BONUS:STAT|CON|4	BONUS:STAT|INT|-8	BONUS:STAT|WIS|6	BONUS:STAT|CHA|-6									BONUS:WEAPONPROF=Swarm|DAMAGE|-STR	ABILITY:Special Ability|AUTOMATIC|Diseased|Distraction|Scent|Sikari Rage	NATURALATTACKS:Swarm,Weapon.Natural.Weapon Group Natural.Melee,*1,2d6														MONSTERCLASS:Animal:10				RACETYPE:Animal	TEMPLATE:Swarm	CR:5			SOURCEPAGE:p.88
+Rajput Ambari		NAMEISPI:YES				SIZE:H	MOVE:Walk,40				REACH:10	VISION:Darkvision (60')	BONUS:HP|CURRENTMAX|112												BONUS:STAT|STR|16	BONUS:STAT|DEX|2	BONUS:STAT|CON|0	BONUS:STAT|INT|0	BONUS:STAT|WIS|0	BONUS:STAT|CHA|-10	BONUS:VAR|AC_Natural_Armor|12|TYPE=Base							ABILITY:Special Ability|AUTOMATIC|Trample|Undead Servitor|War Stomp		NATURALATTACKS:Slam,Weapon.Melee.Natural,*2,2d6|Stamp,Weapon.Melee.Natural,*2,2d6	NATURALATTACKS:Gore,Weapon.Melee.Natural,*1,2d8	DR:10/slashing	MONSTERCLASS:Undead (Mindless):16		RACETYPE:Undead				CR:7			SOURCEPAGE:p.84
+Sikari Macaque Swarm				STARTFEATS:1	SIZE:T	MOVE:Walk,30,Climb,30			REACH:0	VISION:Low-Light Vision					BONUS:SKILL|Climb|16	BONUS:SKILL|Acrobatics|8|TYPE=Racial	BONUS:STAT|STR|-6	BONUS:STAT|DEX|10	BONUS:STAT|CON|4	BONUS:STAT|INT|-8	BONUS:STAT|WIS|6	BONUS:STAT|CHA|-6									BONUS:WEAPONPROF=Swarm|DAMAGE|-STR	ABILITY:Special Ability|AUTOMATIC|Diseased|Distraction|Scent|Sikari Rage	NATURALATTACKS:Swarm,Weapon.Natural.Weapon Group Natural.Melee,*1,2d6														MONSTERCLASS:Animal:10				RACETYPE:Animal	TEMPLATE:Swarm	CR:5			SOURCEPAGE:p.88
```

Comment:
An example with SIZE:T instead of  SIZE:D

Reply:
As above